src/reject-prs.ts

```diff
@@ -24,13 +24,31 @@ import {meowFlags} from './cli';
 import {GitHubRepository, PullRequest} from './lib/github';
 import {processPRs} from './lib/asyncItemIterator';
 
-async function processMethod(repository: GitHubRepository, pr: PullRequest) {
+async function processMethod(
+  repository: GitHubRepository,
+  pr: PullRequest,
+  cli: meow.Result<typeof meowFlags>
+) {
+  const ref = pr.head.ref;
   try {
     await repository.closePullRequest(pr);
   } catch (err) {
     console.warn('    cannot close pull request, skipping:', err.toString());
     return false;
   }
+
+  // delete only branches that we own
+  if (
+    cli.flags.clean &&
+    pr.head.repo.owner.login === repository.repository.owner.login
```

Comment:
This might not really fit our main use case, where branches are created by bots. But in general. is there any harm if we try to delete a branch no matter what, and just ignore the failure? (normally, it does not really matter if a branch was successfully deleted or not, as long as the PR is merged).

Reply:
I made the `--clean` flag so prevent any unexpected behavior changes, but I suppose it could also be the default behavior. In my personal use case I would always want to delete the branch. We would still need to verify the repository owner to prevent accidentally deleting a branch with the same name as a branch in a fork (e.x. if we reject a PR from `prototypicalpro/repo#feature-branch` to `googleapis/repo#master`, we don't want to accidentally delete `googleapis/repo#feature-branch`).